.aws_ci/Jenkinsfile

```diff
@@ -0,0 +1,126 @@
+// Use milestones to abort old builds when the user force pushes
+def buildNumber = env.BUILD_NUMBER as int
+if (buildNumber > 1) milestone(buildNumber - 1)
+milestone(buildNumber)
+
+pipeline {
+    agent {
+        ecs {
+            inheritFrom 'fargate-large'
+        }
+    }
+    options {
+        buildDiscarder(logRotator(daysToKeepStr: "90"))
+        timeout(time: 8, unit: 'HOURS')
+    }
+    environment {
+        // AWS region where the cluster is created
+        REGION="us-west-2"
+    }
+    stages {
+        // Cleanup workspace before job start.
+        stage("Clean up workspace") {
+            steps{
+                deleteDir()
+            }
+        }
+        stage("Checkout SCM repo") {
+            steps {
+                checkout scm
+            }
+        }
+        stage("Download and extract PortaFiducia") {
+            steps {
+                script {
+                    sh 'printenv'
+                    def common = load ".aws_ci/common.groovy"
+                    common.download_and_extract_portafiducia('PortaFiducia')
```

Comment:
is it something we want to disclose?

Reply:
We have to in order to use the multibranch pipeline... I was able to hide the AWS account